manifests/t/Termius/Termius/8.0.4/Termius.Termius.installer.yaml

```diff
@@ -0,0 +1,30 @@
+# Created using wingetcreate 1.2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Termius.Termius
+PackageVersion: 8.0.4
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+Scope: user
+InstallModes:
+- silent
+UpgradeBehavior: install
+Protocols:
+- http
+- ssh
+- telnet
+FileExtensions:
+- pem
+- ppk
+AppsAndFeaturesEntries:
+- DisplayName: Termius 8.0.4
+- DisplayVersion: 8.0.4

```

Comment:
```suggestion
```
These are not needed as PackageName, Publisher are a good match for correlation already